crates/bevy_ui/src/entity.rs

```diff
@@ -24,7 +24,7 @@ pub struct NodeBundle {
     /// Describes the style including flexbox settings
     pub style: Style,
     /// Describes the color of the node
-    pub color: UiColor,
+    pub fill: Fill,
     /// Describes the image of the node
```

Comment:
I would probably have updated the documentation here to something like
`Describes the fill color of the node` ?